misc/CFGs/CFG.txt

```diff
@@ -0,0 +1,87 @@
+**************************************************************
+CFG
+**************************************************************
+
+Program --> Commands $
+
+Commands --> FunctionDcl Commands
```

Comment:
change this into "commands --> command commands" 
and then have command be defined as:
"command --> declaration | assigment | expression | EPSILON